src/docs/ftw-introduction/01-how-to-customize-ftw/index.md

```diff
@@ -1,19 +1,26 @@
 ---
-title: How to Customize FTW
+title: Customizing the template
 slug: how-to-customize-ftw
-updated: 2021-10-19
+updated: 2023-01-01
 category: ftw-introduction
 ingress:
-  So you've decided to build your own marketplace using Flex Template
-  for Web (FTW). That's awesome! This guide will help you in setting up
-  your fork and describes the general workflow.
+  This article helps you set up the development environment and outlines
+  the best practices you should follow while developing on the
+  Sharetribe Web Template.
 published: true
 ---
 
-## Create a marketplace environment
+## Prerequisites
```

Comment:
As long as the Beta release is in use, this section should mention that the new processes need to be added to the marketplace environment. I added something about that to the README.md of the repository:
https://github.com/sharetribe/web-template#take-the-new-beta-processes-into-use

Reply:
Added!